velox/functions/prestosql/tests/URLFunctionsTest.cpp

```diff
@@ -101,6 +100,17 @@ TEST_F(URLFunctionsTest, validateURL) {
   validate("foo", "", "", "", "", "", std::nullopt);
 }
 
+TEST_F(URLFunctionsTest, extractPath) {
+  const auto extractPath = [&](const std::optional<std::string>& url) {
+    return evaluateOnce<std::string>("url_extract_path(c0)", url);
+  };
+
+  ASSERT_EQ(
+      "/media/set/Books and Magazines.php",
+      extractPath(
+          "https://www.cnn.com/media/set/Books%20and%20Magazines.php?foo=bar"));
+}
```

Comment:
Why do we need a function object?
Why not simply:
```
ASSERT_EQ(
      "/media/set/Books and Magazines.php",
      evaluateOnce<std::string>("url_extract_path(c0)",
          "https://www.cnn.com/media/set/Books%20and%20Magazines.php?foo=bar");
```
?

Reply:
To make it easy to add more test cases.

Reply:
Just copy/paste  the ASSERT statement?
The more plain the code is - the easier it is to understand it.